tasks/generate.py

```diff
@@ -35,11 +35,13 @@ def authors(ctx):
 
 
 @invoke.task
-def news(ctx, draft=False):
+def news(ctx, draft=False, yes=False):
```

Comment:
This is an unrelated change. Ideally, this should be in a separate PR.

Reply:
Sure -- I had added this to [include NEWS.rst in the artifacts](https://github.com/pypa/pip/blob/fd490b5da274e6edb0dac1b0dbe2b59df5fc9223/.azure-pipelines/jobs/package.yml#L24), but I can make it just the sdist and wheel for now.

Reply:
Ah, cool cool. I didn't see it in a search of `yes`, so clearly I am bad at doing searches. :P 

I'm ambivalent on this then; no need to change it. :)